bugbot/crash/analyzer.py

```diff
@@ -477,6 +477,41 @@ def is_near_allocator_related_crash(self) -> bool:
         """
         return self.is_near_allocator_crash or self.is_potential_near_allocator_crash
 
+    @property
+    def is_content_crash(self) -> bool:
+        """Whether the crash is related to content process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "content":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_gpu_crash(self) -> bool:
+        """Whether the crash is related to GPU."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "gpu":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_parent_crash(self) -> bool:
+        """Whether the crash is related to the parent process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "parent":
+                return row["count"] > 0
+        return False
+
+    @property
+    def crash_kind(self) -> str:
+        """The kind of the crash based on the crashing process type."""
+        if self.is_plugin_crash:
+            return "Plugin"
+        if self.is_parent_crash:
+            return "Parent"
+        if self.is_content_crash:
```

Comment:
Is there a reason you are looking for a few specific types of processes and not just returning the value of the field (like sanitization, say)? In any event, looking at crash-stats, we also have GPU, RDD, Socket and Utility crashes, so those should be included in here somehow. Having a big switch statement would be a bit cleaner for adding a bunch of new process types, IMO.

Reply:
I just focused on the types that you mentioned in your feedback: https://github.com/mozilla/bugbot/issues/2185#issue-1822737043. If the goal is proved the most frequent process type for the crash signature, we could print out the first time in the list (the types are sorted based ont he frequency): 
```py
self.signature["facets"]["process_type"][0]["term"]
```